### ISSUE TITLE
mod only loads in Java 8

### ISSUE THREAD

**Issue by calog3no** [TimeConqueror/Time-Core#3]
I attempted to run an modpack instance that contains the time core mod in it. (Rustic Waters II) and I would get a crash during mod loading with the error that time core would not load. I was using the gd Launcher and had my Java set to 14, once I switched the Java ver to 8 I was able to load and run the pack with no issues.

**Comment by Quipex** [comment]
Same thing (Rustic Waters II). Thanks that pointed out it was due to java other than 8. Once switched to 8, it has loaded fine.

The crashlog:
```
[00:43:03] [Render thread/FATAL] [ne.mi.fm.ModLoader/]: Encountered non-modloading exceptions!
java.util.concurrent.CompletionException: java.lang.RuntimeException
	at java.util.concurrent.CompletableFuture.encodeRelay(CompletableFuture.java:367) ~[?:?] {}
	at java.util.concurrent.CompletableFuture.completeRelay(CompletableFuture.java:376) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1148) ~[?:?] {}
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506) ~[?:?] {}
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:610) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1159) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:479) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598) ~[?:?] {re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183) ~[?:?] {}
Caused by: java.lang.RuntimeException
	at net.minecraftforge.fml.ModList.completableFutureFromExceptionList(ModList.java:125) ~[forge:?] {re:classloading}
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1146) ~[?:?] {}
	... 9 more
	Suppressed: java.lang.ExceptionInInitializerError
		at ru.timeconqueror.timecore.api.reflection.provider.JavaClassHandler.findMethod(JavaClassHandler.java:30) ~[timecore:1.16.5-3.4.1.0] {re:classloading}
		at ru.timeconqueror.timecore.misc.ModInitializer.processTimeAutoRegInitMethod(ModInitializer.java:108) ~[timecore:1.16.5-3.4.1.0] {re:mixin,re:classloading}
		at ru.timeconqueror.timecore.misc.ModInitializer.lambda$setupAutoRegistries$1(ModInitializer.java:70) ~[timecore:1.16.5-3.4.1.0] {re:mixin,re:classloading}
		at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?] {}
		at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?] {}
		at java.util.Iterator.forEachRemaining(Iterator.java:133) ~[?:?] {}
		at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801) ~[?:?] {}
		at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?] {}
		at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?] {}
		at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?] {}
		at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?] {}
		at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?] {}
		at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?] {}
		at ru.timeconqueror.timecore.misc.ModInitializer.setupAutoRegistries(ModInitializer.java:57) ~[timecore:1.16.5-3.4.1.0] {re:mixin,re:classloading}
		at ru.timeconqueror.timecore.misc.ModInitializer.run(ModInitializer.java:39) ~[timecore:1.16.5-3.4.1.0] {re:mixin,re:classloading}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.handler$znn000$fmlModConstructingHook(FMLModContainer.java:532) ~[forge:36.2] {re:mixin,re:classloading,pl:mixin:APP:mixins.timecore.json:FMLModContainerMixin,pl:mixin:A}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:83) ~[forge:36.2] {re:mixin,re:classloading,pl:mixin:APP:mixins.timecore.json:FMLModContainerMixin,pl:mixin:A}
		at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[forge:?] {re:classloading,re:mixin}
		at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1800) ~[?:?] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1792) ~[?:?] {}
		at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665) ~[?:?] {re:computing_frames}
		at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598) ~[?:?] {re:computing_frames}
		at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183) ~[?:?] {}
	Caused by: java.lang.RuntimeException: Can't retrieve field modifiers from class class java.lang.reflect.Field
		at ru.timeconqueror.timecore.api.reflection.ReflectionHelper.findField(ReflectionHelper.java:117) ~[timecore:1.16.5-3.4.1.0] {re:classloading}
		at ru.timeconqueror.timecore.api.reflection.ReflectionHelper.<clinit>(ReflectionHelper.java:16) ~[timecore:1.16.5-3.4.1.0] {re:classloading}
		... 25 more
	Caused by: java.lang.NoSuchFieldException: modifiers
		at java.lang.Class.getDeclaredField(Class.java:2549) ~[?:?] {}
		at ru.timeconqueror.timecore.api.reflection.ReflectionHelper.findField(ReflectionHelper.java:114) ~[timecore:1.16.5-3.4.1.0] {re:classloading}
		at ru.timeconqueror.timecore.api.reflection.ReflectionHelper.<clinit>(ReflectionHelper.java:16) ~[timecore:1.16.5-3.4.1.0] {re:classloading}
		... 25 more
	Suppressed: java.lang.NoClassDefFoundError: Could not initialize class ru.timeconqueror.timecore.api.reflection.ReflectionHelper
		at ru.timeconqueror.timecore.misc.ModInitializer.processAutoRegistrable(ModInitializer.java:80) ~[timecore:1.16.5-3.4.1.0] {re:mixin,re:classloading}
		at ru.timeconqueror.timecore.misc.ModInitializer.lambda$setupAutoRegistries$1(ModInitializer.java:68) ~[timecore:1.16.5-3.4.1.0] {re:mixin,re:classloading}
		at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?] {}
		at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?] {}
		at java.util.Iterator.forEachRemaining(Iterator.java:133) ~[?:?] {}
		at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801) ~[?:?] {}
		at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?] {}
		at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?] {}
		at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?] {}
		at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?] {}
		at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?] {}
		at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?] {}
		at ru.timeconqueror.timecore.misc.ModInitializer.setupAutoRegistries(ModInitializer.java:57) ~[timecore:1.16.5-3.4.1.0] {re:mixin,re:classloading}
		at ru.timeconqueror.timecore.misc.ModInitializer.run(ModInitializer.java:39) ~[timecore:1.16.5-3.4.1.0] {re:mixin,re:classloading}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.handler$znn000$fmlModConstructingHook(FMLModContainer.java:532) ~[forge:36.2] {re:mixin,re:classloading,pl:mixin:APP:mixins.timecore.json:FMLModContainerMixin,pl:mixin:A}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:83) ~[forge:36.2] {re:mixin,re:classloading,pl:mixin:APP:mixins.timecore.json:FMLModContainerMixin,pl:mixin:A}
		at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:106) ~[forge:?] {re:classloading,re:mixin}
		at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1800) ~[?:?] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1792) ~[?:?] {}
		at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016) ~[?:?] {}
		at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665) ~[?:?] {re:computing_frames}
		at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598) ~[?:?] {re:computing_frames}
		at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183) ~[?:?] {}
```

**Comment by TimeConqueror** [comment]
Fixed in 3.5.0.0! Thanks for reporting!
